EasyPost.Net/EasyPost.Net.csproj

```diff
@@ -45,7 +49,11 @@
     </None>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="11.0.2"/>
-    <PackageReference Include="RestSharp" Version="106.15.0"/>
+    <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
+    <PackageReference Include="RestSharp" Version="107.3.0" />
   </ItemGroup>
+
+  <ItemGroup>
+  </ItemGroup>
```

Comment:
Let's remove this empty group

Reply:
Resolved in [d903ec7](https://github.com/EasyPost/easypost-csharp/pull/214/commits/d903ec7476be02f732e6eb4245a85595fc72df59)